:pencil2: fix some typos and update local execution (#68)
massalabs/massa-docs: web3-dev/dummy-network-generation.rst

    cd massa-client && cargo run node_add_staking_secret_keys SECRETK

> The network with your node all start in 10 seconds and you can now interact it with

Reviewer: ```suggestion
The local network and your node will start after 10 seconds. Once it is started, can now interact with it using
```

> The local network and your node will start after 10 seconds. Once it is started, can now interact with it using

Author: ```suggestion
The local network and your node will start after 10 seconds. Once it is started, you can interact with it using
```

> the CLI client like a testnet node.

Reviewer: ```suggestion
the CLI client as you would with a testnet node.
```
